### ISSUE TITLE
about:brave copy to clipboard tooltip is not working

### ISSUE THREAD

**Issue by Liunkae** [brave/browser-laptop#8199]
## Test Plan:
1. Open about:brave with kabob > Help > About Brave.
2. Hover over the copy to clipboard button as shown in the screenshot below.
3. Make sure the tooltip `Copy to clipboard` tooltip is shown.

## Original issue description
- Did you search for similar issues before submitting this one?
Yes.

- Describe the issue you encountered:
On the about:brave page, if you hover over the copy to clipboard button, it does not show the tooltip.

- Platform (Win7, 8, 10? macOS? Linux distro?):
All.

- Brave Version (revision SHA):
0.14.1 (3de60d5)

- Steps to reproduce:
    1. Open about:brave with kabob > Help > About Brave.
    2. Hover over the copy to clipboard button as shown in the screenshot below.
    
- Actual result:
No tooltip is shown.

- Expected result:
The tooltip `Copy to clipboard` should be shown.

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes.

- Is this an issue in the currently released version?
Yes, 0.14.1.

- Can this issue be consistently reproduced?
Yes.

- Extra QA steps:
    1. Open about:brave with kabob > Help > About Brave.
    2. Hover over the copy to clipboard button as shown in the screenshot below.
    3. Make sure the tooltip `Copy to clipboard` tooltip is shown.

- Screenshot if needed:
![image](https://cloud.githubusercontent.com/assets/19424103/24875672/82278c8e-1dee-11e7-92d8-218ce026ae0b.png)

- Any related issues:
None.
